internal/faustjs.org/docs/migrationPath/overview.mdx

```diff
@@ -2,6 +2,268 @@
+
+<img
+  src="/docs/img/possible-templates.png"
+  alt="The list of possible templates as shown in the dev console"
+/>
+
+You can see an example of the hierarchy in the [blueprint project](https://github.com/wpengine/atlas-blueprint-portfolio/tree/main/wp-templates) where we provided implementations for the main templates.
+
+## Overview of the changes needed in to migrate from GQty to Apollo
+
+Given the above information, we can now list an brief overview of the changes needed in to migrate from GQty to Apollo.
+You can also review the code of this [Blueprint Project](https://github.com/wpengine/atlas-blueprint-portfolio/tree/main) that shows how the end result will look like.
+
+**1. Capture and migrate Presentational Components**
+
+Create a list of components that can be reused into the new version of Faust and move them into the new projects folder.
+
+**2. Import Faust Dependencies**
+
+Start by importing the Faust dependencies:
+
+`npm i @faustwp/cli @faustwp/core graphql @apollo/client`
+
+Change the `package.json` scripts to use the new faust cli:
+
+```json title="package.json"
+scripts": {
+    "dev": "faust dev",
+    "build": "faust build",
+    "generate": "faust generatePossibleTypes",
+    "start": "faust start",
+    "lint": "faust lint"
+	...
+}
+```
+
+**3. Use new Faust Configuration**
+
+Replace the `withFaust` configuration function with the new one:
+
+For the `next.config.js`:
+
+```js title="next.config.js"
+// const { withFaust } = require('@faustjs/next');
+const { withFaust, getWpHostname } = require('@faustwp/core');
+```
+
+For the `faust.config.js`:
+
+```js title="faust.config.js"
+import { setConfig } from '@faustwp/core';
+import templates from './wp-templates';
+import possibleTypes from './possibleTypes.json';
+
+/**
+ * @type {import('@faustwp/core').FaustConfig}
+ **/
+export default setConfig({
+  templates,
+  experimentalPlugins: [],
+  possibleTypes,
+});
+```
+
+**4. Use new Faust Imports and Providers**
+Replace old Faust references with the new ones:
+
+For the api route:
+
+```js title="pages/api/faust/[[...route]].js"
+// import { apiRouter } from '@faustjs/core/api';
+import { apiRouter } from '@faustwp/core';
+```
+
+For the `_app.js`:
+
+```js title="_app.js"
+import '../faust.config';
+import React from 'react';
+import { useRouter } from 'next/router';
+import { FaustProvider } from '@faustwp/core';
+import '../styles/main.scss';
+
+export default function MyApp({ Component, pageProps }) {
+  const router = useRouter();
+
+  return (
+    <>
+      <FaustProvider pageProps={pageProps}>
+        <Component {...pageProps} key={router.asPath} />
+      </FaustProvider>
+    </>
+  );
+}
+```
+
+**5. Create the new wp-templates folder and include an index**
+The blueprint example project showcases how the components are structured and it's a great way to understand what needs to be done.
```

Comment:
```suggestion
:::tip
The blueprint example project showcases how the components are structured and is a great way to understand what changes to make during this step.
:::

```

Reply:
A great suggestion! Is this Faust's blank scaffold Blake is working on this sprint? Or the portfolio?

Reply:
If we can link to whichever blueprint this one references, I think that will be really helpful for clarity